entrypoint.sh

```diff
@@ -1,6 +1,16 @@
 #!/usr/bin/env bash
 geoserver-plugin-download.sh ${CATALINA_BASE}/webapps/${APP_LOCATION}/WEB-INF/lib $PLUGIN_DYNAMIC_URLS
 set -m
+
+# FIXME: this workaround (for a deploy in a Swarm) keeps GS from starting and
+# crashing, because of JDBC not being able to connect to Postgres
+db_err=6
+while [ $db_err -eq 6 ]; do
+  curl -sS "$DATABASE_HOST"
```

Comment:
Does this work with TCP port 5432?

Reply:
also consider using this script: https://github.com/geosolutions-it/docker-geoserver/blob/master/docker-compose.yml#L11

Reply:
No. Here I'm only testing the hostname resolution. curl exits with 6 in case of DNS resolution failure.